x-pack/qa/runtime-fields/src/main/java/org/elasticsearch/xpack/runtimefields/test/CoreTestTranslater.java

```diff
@@ -222,10 +222,32 @@ public boolean modifySections(List<ExecutableSection> executables) {
          */
         protected abstract boolean modifySearch(ApiCallSection search);
 
+        private static Object getSetting(final Object map, final String... keys) {
```

Comment:
Why is this change needed?

Reply:
This is required to skip testing using runtime fields when `ignore_above` is used. This is done already when `ignroe_above` is used at mapping level (runtime fields do not support `ignroe_above`). 
See this code below:
```
if (propertyMap.containsKey("ignore_above")) {
                    // Scripts don't support ignore_above so we skip those fields
                    continue;
                }
```

Now, since the `ignore_above` is at index-level is is no longer at field mapping level and so tests are not skipped.